sapl-spring-data-mongo-reactive/src/main/java/io/sapl/springdatamongoreactive/sapl/queries/enforcement/MongoAnnotationQueryManipulationEnforcementPoint.java

```diff
@@ -48,7 +49,7 @@
  * @param <T> is the domain type.
  */
 public class MongoAnnotationQueryManipulationEnforcementPoint<T> implements QueryManipulationEnforcementPoint<T> {
-    private static final String MONGO_QUERY_MANIPULATION = "mongoQueryManipulation";
+    private static final String mongoQueryManipulation = "mongoQueryManipulation";
```

Comment:
upper case.
Please, you keep switching up the standard Java conventions regarding the notation of variable names. 

constants are ALWAYS_WRITTEN_THIS_WAY
I already fixed this for you and you reversed it. Doble and triple-check your code base.